apps/web/src/scenes/CommunityPage/CommunityMetadataFormModal.tsx

```diff
@@ -0,0 +1,136 @@
+import { BaseM } from '~/components/core/Text/Text';
+import { TextArea } from '~/components/core/TextArea/TextArea';
+import { useModalActions } from '~/contexts/modal/ModalContext';
+import { useToastActions } from '~/contexts/toast/ToastContext';
+import { CommunityMetadataFormModalFragment$key } from '~/generated/CommunityMetadataFormModalFragment.graphql';
+import colors from '~/shared/theme/colors';
+
+type Props = {
+  communityRef: CommunityMetadataFormModalFragment$key;
+};
+
+export function CommunityMetadataFormModal({ communityRef }: Props) {
+  const community = useFragment(
+    graphql`
+      fragment CommunityMetadataFormModalFragment on Community {
+        dbid
+        name
+      }
+    `,
+    communityRef
+  );
+
+  const { hideModal } = useModalActions();
+  const { pushToast } = useToastActions();
+
+  const [message, setMessage] = useState<string>('');
+  const [status, setStatus] = useState<'SUBMITTING' | 'SUCCESS' | 'ERROR' | undefined>();
+
+  const handleSubmit = useCallback(async () => {
+    if (!message) return;
+    setStatus('SUBMITTING');
+
+    try {
+      const response = await fetch(
+        `https://formspree.io/f/${process.env.NEXT_PUBLIC_FORMSPEE_REQUEST_COLLECTION_ID}`,
+        {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            communityId: community.dbid,
+            communityName: community.name,
+            message,
+          }),
+        }
+      );
+
+      if (response.status === 200) {
+        setStatus('SUCCESS');
+        pushToast({
+          message: 'Submitted successfully!',
+        });
+        hideModal();
+      } else {
+        setStatus('ERROR');
+      }
+    } catch (error) {
+      setStatus('ERROR');
+    }
+  }, [community.dbid, community.name, message, pushToast, hideModal]);
+
+  return (
+    <StyledConfirmation gap={16}>
+      <VStack gap={8}>
+        <TextContainer>
+          <StyledBaseM>
+            We're currently updating our editor for collection pages. Let us know what changes you’d
+            like and we'll jump on them right away!
+          </StyledBaseM>
+        </TextContainer>
+        <StyledTextAreaWrapper isError={status === 'ERROR'}>
+          <TextArea
+            value={message}
+            onChange={(e) => setMessage(e.target.value)}
+            placeholder="Update Title, Description and/or Profile Picture (include image link)"
+          />
+        </StyledTextAreaWrapper>
+        {status === 'ERROR' && (
+          <StyledErrorText>Something went wrong! Please try again</StyledErrorText>
+        )}
+      </VStack>
+      <ButtonContainer>
+        <Button
+          eventElementId="Community Metadata Form Submit Button"
+          eventName="Submit Community Metadata Form"
+          eventContext={contexts.Community}
+          onClick={handleSubmit}
+        >
+          Submit
```

Comment:
oo could you disable this button if there's no input? (ie disabled by default, becomes enabled when the user types)

Reply:
Updated!